lib/lightning/attempts.ex

```diff
@@ -107,9 +107,9 @@ defmodule Lightning.Attempts do
   end
 
   def start_attempt(%Attempt{} = attempt) do
-    Attempt.start(attempt)
-    |> update_attempt()
-    |> track_attempt_queue_delay()
+    with {:ok, attempt} <- Attempt.start(attempt) |> update_attempt() do
```

Comment:
My appologies @rorymckinley , I see there is change on the returned attempt. So instead of a `with`, a `tap` would provide a better semantics in this case, without changing the result of `update_attempt`.